packages/tupaia-web/src/features/Dashboard/Dashboard.tsx

```diff
@@ -29,7 +29,7 @@ const Panel = styled.div<{
   $isExpanded: boolean;
 }>`
   position: relative;
-  background-color: ${({ theme }) => theme.panel.background};
+  background-color: ${({ theme }) => theme.searchBar.background};
```

Comment:
This works, but to be really pedantic I would maybe make a theme variable called something more generic, like `backgrounds` that would have `primary` and `secondary` inside instead of using `searchBar` and then everywhere that refers to `searchBar.background` would instead use `backgrounds.primary` and for the lighter black it would refer to `backgrounds.secondary`. Then if we ever want to redo the theme it's clear what we are updating